src/pages/install/{installConfig.agentName}.js

```diff
@@ -14,6 +16,14 @@ import AppInfoConfig from '../../components/AppInfoConfig';
 import AppInfoConfigOption from '../../components/AppInfoConfigOption';
 import InstallNextSteps from '../../components/InstallNextSteps';
 
+const slugify = (str) =>
+  str
+    .trim()
+    .toLowerCase()
+    .replace(/\s+/g, '-')
+    .replace(/-+/, '-')
+    .replace(/[^a-z0-9-]/g, '');
```

Comment:
we do have this as an exportable function in src/scripts/utils if you wanna import instead

Reply:
the import was being weird - I think bc of ESM vs CommonJs modules or something like that